styles/xrpl.scss

```diff
@@ -71,7 +71,6 @@ $line-height-base: 1.5;
 @import "_feedback.scss";
 @import "_video.scss";
 @import "_contribute.scss";
-// @import "_top-banner.scss";
```

Comment:
Removed this redundant import (#2479 added a new one on line 54 instead of uncommenting this one)